tests/gitea_release/release_assets/dry_run_output.txt

```diff
@@ -0,0 +1,5 @@
+Would create a release on Gitea with name 1.1.0 ([DATE]) and tag v1.1.0 and body:
```

Comment:
Think we should put the host here instead of `Gitea`?

Reply:
mmmmh... yeah that makes sense to me